README.md

```diff
@@ -394,6 +394,37 @@ Add a plugin as a dependency, so it's installed locally:
 
 </details>
 
+<details>
+ <summary>Output usage</summary><br>
```

Comment:
Looks boring :)  any ideas on better title ?

Reply:
**Use `output` for a powerful composition with other actions**